src/System.Security.Cryptography.X509Certificates/tests/ChainTests.cs

```diff
@@ -13,9 +13,7 @@ public static class ChainTests
     {
         // #9293: Our Fedora and Ubuntu CI machines use NTFS for "tmphome", which causes our filesystem permissions checks to fail.
         internal static bool IsReliableInCI { get; } =
-            !PlatformDetection.IsFedora24 &&
-            !PlatformDetection.IsFedora25 &&
-            !PlatformDetection.IsFedora26 &&
+            !PlatformDetection.IsFedora &&
```

Comment:
I left them just in case will be needed later. but I can remove IsFedora24 as we are supporting version 25 and up now. 

@danmosemsft can you confirm please